web/src/app/chat/lib.tsx

```diff
@@ -698,7 +698,8 @@ export async function useScrollonStream({
       ) {
         // catch up if necessary!
         const scrollAmount = scrollDist.current + 10000;
-        if (scrollDist.current > 140) {
+        if (scrollDist.current > 300) {
```

Comment:
Would be nice to refactor out these magic numbers.

Reply:
Agreed!